src/plugins/wizard/public/application/components/data_tab/secondary_panel.tsx

```diff
@@ -4,15 +4,15 @@
  */
 
 import React, { useCallback, useMemo, useState } from 'react';
-import { cloneDeep } from 'lodash';
+import { cloneDeep, get } from 'lodash';
```

Comment:
boo to lodash methods 😉

Reply:
I blame typescript. I gave into this after fighting with typescript to get the same behavior for this:

```ts
const indexField = availableFields.find(
  ({ name }) => name === get(draftAgg, 'params.field')
);
```